x/tss/keeper/keeper.go

```diff
@@ -348,7 +347,7 @@ func (k Keeper) GetAvailableOperators(ctx sdk.Context, keyIDs ...exported.KeyID)
 		}
 
 		if !k.operatorHasKeys(ctx, address, keyIDs...) {
-			k.Logger(ctx).Debug(fmt.Sprintf("excluding validator %s due absent keys", validator))
+			k.Logger(ctx).Debug(fmt.Sprintf("excluding validator %s due to absent keys", validator))
```

Comment:
don't want to make this info level?

Reply:
Info/Debug is not particularly useful rn unless we need to show on dashboard. Logging needs to be revisited in general.